src/tools/core/pcp_combine/pcp_combine.cc

```diff
@@ -1546,6 +1626,9 @@ void usage() {
         << "variable names to be written to the \"out_file\" "
         << "(optional).\n"
 
+        << "\t\t\"-input_thresh\" overrides the default required ratio "
+        << "of valid input files (" << input_thresh << ") (optional).\n"
+
         << "\t\t\"-vld_thresh\" overrides the default required ratio "
         << "of valid data (" << vld_thresh << ") (optional).\n"
```

Comment:
Consider refining this to make it more clear the distinction between input and vld thresh
```suggestion
        << "of valid data at a grid point (" << vld_thresh << ") (optional).\n"
```

Reply:
Updated via a change to my branch. Thanks.